### ISSUE TITLE
Twitch AttributeError

### ISSUE THREAD

**Issue by krathalan** [streamlink/streamlink#3066]
## Bug Report

- [x] This is a bug report and I have read the contribution guidelines.

### Description

When I try to use streamlink to watch a Twitch stream, streamlink doesn't work. Streamlink spits out a Traceback with a final "AttributeError".

### Expected / Actual behavior

Streamlink should be able to watch Twitch streams -- commands which were working a few days ago do not work now.

### Reproduction steps / Explicit stream URLs to test

1. Try to open any Twitch stream
2. `streamlink https://www.twitch.tv/ifrostbolt best`
3. `streamlink https://www.twitch.tv/supertf best`

### Log output

```
[cli][debug] OS:         Linux-5.7.7-arch1-1-x86_64-with-glibc2.2.5
[cli][debug] Python:     3.8.3
[cli][debug] Streamlink: 1.4.1
[cli][debug] Requests(2.24.0), Socks(1.7.1), Websocket(0.57.0)
[cli][info] Found matching plugin twitch for URL https://www.twitch.tv/ifrostbolt
[plugin.twitch][debug] Getting live HLS streams for ifrostbolt
Traceback (most recent call last):
  File "/usr/sbin/streamlink", line 11, in <module>
    load_entry_point('streamlink==1.4.1', 'console_scripts', 'streamlink')()
  File "/usr/lib/python3.8/site-packages/streamlink_cli/main.py", line 1024, in main
    handle_url()
  File "/usr/lib/python3.8/site-packages/streamlink_cli/main.py", line 585, in handle_url
    streams = fetch_streams(plugin)
  File "/usr/lib/python3.8/site-packages/streamlink_cli/main.py", line 464, in fetch_streams
    return plugin.streams(stream_types=args.stream_types,
  File "/usr/lib/python3.8/site-packages/streamlink/plugin/plugin.py", line 317, in streams
    ostreams = self._get_streams()
  File "/usr/lib/python3.8/site-packages/streamlink/plugins/twitch.py", line 794, in _get_streams
    return self._get_hls_streams("live")
  File "/usr/lib/python3.8/site-packages/streamlink/plugins/twitch.py", line 733, in _get_hls_streams
    streams = TwitchHLSStream.parse_variant_playlist(
  File "/usr/lib/python3.8/site-packages/streamlink/stream/hls.py", line 405, in parse_variant_playlist
    locale = session_.localization
  File "/usr/lib/python3.8/site-packages/streamlink/session.py", line 514, in localization
    return Localization(self.get_option("locale"))
  File "/usr/lib/python3.8/site-packages/streamlink/utils/l10n.py", line 122, in __init__
    self.language_code = language_code
  File "/usr/lib/python3.8/site-packages/streamlink/utils/l10n.py", line 147, in language_code
    self.language, self.country = self._parse_locale_code(language_code)
  File "/usr/lib/python3.8/site-packages/streamlink/utils/l10n.py", line 132, in _parse_locale_code
    return self.get_language(parts[0]), self.get_country(parts[1])
  File "/usr/lib/python3.8/site-packages/streamlink/utils/l10n.py", line 176, in get_language
    return Language.get(language)
  File "/usr/lib/python3.8/site-packages/streamlink/utils/l10n.py", line 76, in get
    return Language(lang.alpha_2, lang.alpha_3, lang.name, getattr(lang, "bibliographic", None))
  File "/usr/lib/python3.8/site-packages/pycountry/db.py", line 19, in __getattr__
    raise AttributeError
AttributeError
```

**Comment by krathalan** [comment]
It seems this issue is solved by using `streamlink-git` from the AUR. I searched open and closed issues but couldn't find something similar. I assume this issue is fixed in the next release? Hoping for one soon :) Appreciate the hard work.

**Comment by bastimeyer** [comment]
Duplicate of #3058 and #3062
Fixed here: #3057
